### ISSUE TITLE
Alias Datenpunkt JavaScript Knovertierungsfunktion mit Binding

### ISSUE THREAD

**Issue by Erwin40** [ioBroker/ioBroker#589]
Hallo erst Mal.

Gibt es irgendeine Möglichkeit innerhalb der Konvertierungsfunktion zusätzlich auf einen weiteren Datenpunkt per Binding, zuzugreifen?
So was in der Art `val + {zigbee.0.a4c138b202fed075.temperature_calibration}`

Ich habe nämlich einen Temperatursensor, bei dem in einem weiteren DP ein Kalibrierungswert hinterlegt werden kann.
Leider korrigiert der ZigBee Adapter die gemessene Temperatur nicht mit diesem Wert und ich muss es extra per zusätzl. JavaScript umrechnen.
Hierfür wäre aber doch der Alias DP eine elegante Lösung, wenn ich nur Zugriff auf den zusätzl. DP hätte :-)

**Comment by Apollon77** [comment]
Nein, dies ist aktuell nicht möglich. Ist also eher ein Thema für ein kleines Skriptchen
